src/main/java/uk/gov/hmcts/reform/civil/model/CaseDataLip.java

```diff
@@ -0,0 +1,18 @@
+package uk.gov.hmcts.reform.civil.model;
+
```

Comment:
This file is not in the correct path, you are missing a citizenui folder on the last level. I remember commenting it when you did the demo.

This is the correct path to the file: 

`src/main/java/uk/gov/hmcts/reform/civil/model/citizenui/CaseDataLiP.java`

Reply:
I have removed it, the file already exists.